addons/pos_sale/models/pos_order.py

```diff
@@ -14,3 +14,9 @@ def _compute_currency_rate(self):
         for order in self:
             date_order = order.date_order or fields.Datetime.now()
             order.currency_rate = self.env['res.currency']._get_conversion_rate(order.company_id.currency_id, order.pricelist_id.currency_id, order.company_id, date_order)
+
+    @api.multi
+    def _prepare_invoice(self):
+        invoice_vals = super(PosOrder, self)._prepare_invoice()
```

Comment:
```suggestion
        invoice_vals = super()._prepare_invoice()
```
Reference https://github.com/odoo/odoo/pull/33975#discussion_r291676373